examples/quickbooks/quickbooks.js

```diff
@@ -0,0 +1,68 @@
+const { Reshuffle, SQLStoreAdapter, MemoryStoreAdapter } = require('reshuffle')
+const { QuickbooksConnector } = require('reshuffle-quickbooks-connector')
+const { Pool } = require('pg')
+
+// This example uses reshuffle-quickbooks-connector
+// Code and documentation available on Github: https://github.com/reshufflehq/reshuffle-quickbooks-connector
+
+// Create an API token from your Shopify Admin Page:
+// 1. Go to https://<your_shop_name>.myshopify.com/admin/apps/private
+// 2. Click `Create new private app`,
+// 3. Provide a name and scopes (keeping in mind that permissions are very limited per default)
+// 4. Click Save, and you'll get api key and password
+
+const app = new Reshuffle()
+// Database Store:
+// const pool = new Pool({user: 'RESHUFFLE_DB_USER',
+//   host: 'RESHUFFLE_DB_HOST',
+//   database: 'RESHUFFLE_DB',
+//   password: 'RESHUFFLE_DB_PASS',
+//   port: RESHUFFLE_DB_PORT})
+// const persistentStore = new SQLStoreAdapter(pool, 'reshuffledb')
+
+const persistentStore = new MemoryStoreAdapter()
```

Comment:
I think it is the default and we don't even need to do app.setPersistentStore(new MemoryStoreAdapter()).